ios/GoogleService-Info.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>261471341992-53ajr0r3jgn8gq39ll8dhn00krjo9btu.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.261471341992-53ajr0r3jgn8gq39ll8dhn00krjo9btu</string>
+	<key>API_KEY</key>
+	<string>AIzaSyDTy1YuFh8YC7pUAPy_zxj-OJqZ8qGPKOY</string>
+	<key>GCM_SENDER_ID</key>
+	<string>261471341992</string>
+	<key>PLIST_VERSION</key>
+	<string>1</string>
+	<key>BUNDLE_ID</key>
+	<string>com.keepCoding.home-trainer</string>
+	<key>PROJECT_ID</key>
+	<string>home-trainer-v10</string>
+	<key>STORAGE_BUCKET</key>
+	<string>home-trainer-v10.appspot.com</string>
+	<key>IS_ADS_ENABLED</key>
+	<false></false>
+	<key>IS_ANALYTICS_ENABLED</key>
+	<false></false>
+	<key>IS_APPINVITE_ENABLED</key>
+	<true></true>
+	<key>IS_GCM_ENABLED</key>
+	<true></true>
+	<key>IS_SIGNIN_ENABLED</key>
+	<true></true>
+	<key>GOOGLE_APP_ID</key>
+	<string>1:261471341992:ios:5f9a6197b91c49a1e5619d</string>
+</dict>
+</plist>
```

Comment:
this file should be a .gitignore file

Reply:
Should I put every validation in separate files or make a single file with all the validations?

Reply:
This file alow me to interact with Fierbase, why should it be .gitignore?